src/sound/soundbase.cpp

```diff
@@ -232,98 +221,176 @@ QVector<QString> CSoundBase::LoadAndInitializeFirstValidDriver ( const bool bOpe
 }
 
 /******************************************************************************\
-* MIDI handling                                                                *
+* Command Line Handling                                                        *
 \******************************************************************************/
 void CSoundBase::ParseCommandLineArgument ( const QString& strMIDISetup )
 {
+    if ( strMIDISetup.isEmpty() )
+        return;
+
     int iMIDIOffsetFader = 70; // Behringer X-TOUCH: offset of 0x46
 
-    // parse the server info string according to definition: there is
-    // the legacy definition with just one or two numbers that only
-    // provides a definition for the controller offset of the level
+    // default to the legacy kind of specifying
+    // the fader controller offset without an
+    // indication of the count of controllers
+    bool bSimple = true;
+
+    // Parse the --ctrlmidich string.  There are two formats.
+    //
+    // The legacy definition has just one or two numbers that only
```

Comment:
I vote this comment is removed.  It doesn't explain the code, only provides a "user guide" or "how-to" for someone using the feature.  So it's better in user-facing documentation.

Reply:
Done.

https://github.com/jamulussoftware/jamuluswebsite/pull/1053 raised to update the documentation.